main.tf

```
@@ -1 +1,654 @@
-# Your Terraform code goes here :)
+locals {
+  agentless_scan_ecs_task_role_arn = var.global ? aws_iam_role.agentless_scan_ecs_task_role[0].arn : ""
+  agentless_scan_ecs_execution_role_arn = var.global ? aws_iam_role.agentless_scan_ecs_execution_role[0].arn : ""
+  agentless_scan_ecs_event_role_arn = var.global ? aws_iam_role.agentless_scan_ecs_event_role[0].arn : ""
```

If we're not deploying the global infrastructure, we need to pass through the provided ECS roles, correct?

```suggestion
  agentless_scan_ecs_task_role_arn = var.global ? aws_iam_role.agentless_scan_ecs_task_role[0].arn : var.agentless_scan_ecs_task_role_arn
  agentless_scan_ecs_execution_role_arn = var.global ? aws_iam_role.agentless_scan_ecs_execution_role[0].arn : var.agentless_scan_ecs_execution_role_arn
  agentless_scan_ecs_event_role_arn = var.global ? aws_iam_role.agentless_scan_ecs_event_role[0].arn : var.agentless_scan_ecs_event_role_arn
```